pkg/proxy/proxy_test.go

```diff
@@ -278,16 +278,17 @@ func TestUpdateMissingHelmRelease(t *testing.T) {
 	ch := &chart.Chart{
 		Metadata: &chart.Metadata{Name: chartName, Version: version},
 	}
-	// Simulate the same app but in a different namespace
+
 	ns2 := "other_ns"
-	app := AppOverview{rs, version, ns2, "icon.png", "DEPLOYED"}
+	rs2 := "not_foo"
+	app := AppOverview{rs2, version, ns2, "icon.png", "DEPLOYED"}
 	proxy := newFakeProxy([]AppOverview{app})
 
 	_, err := proxy.UpdateRelease(rs, ns, "", ch)
 	if err == nil {
 		t.Error("Update should fail, there is not a release in the namespace specified")
 	}
-	if !strings.Contains(err.Error(), "not found") {
+	if !strings.Contains(err.Error(), "No such release") {
```

Comment:
because this test was failing before at the end of the function (line proxy.go:109) since the namespace was different (but not the name). That no longer fails there because we are not manually checking the namespace (we rely in the helm library to behave). The only way to make this fail with the new code is to use a name that doesn't exist. In that case the function fails at proxy.go:99.

Reply:
I see, makes sense!